site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -262,9 +262,27 @@ The unique id of the signing key used to sign the access token.
 [field]#{base_field_name}.jwtConfiguration.idTokenKeyId# [type]#[UUID]# [since]#Available since 1.8.0#::
 The unique id of the signing key used to sign the Id token.
 
+[field]#{base_field_name}.jwtConfiguration.refreshTokenExpirationPolicy# [type]#[String]# [since]#Available since 1.17.0#::
+The refresh token expiration policy. The following are valid values:
+
+* ``Fixed`` - the expiration is calculated from the time the token is issued.
+* ``SlidingWindow`` - the expiration is calculated from the last time the token was used.
```

Comment:
Did you intend to use double back ticks?

Reply:
Thought I was being consistent. See `site/docs/v1/tech/apis/_transaction-types.adoc` / SimpleMajority. 

On looking around, single backticks seem more consistent, will change.

Reply:
Interesting, I tried removing the double back ticks on that one, no change in render. I wonder if that was either a mistake, or perhaps added due to how it is being included or something like that.